Assets/Animations/Effect/Item Pickup/Item Pickup.controller

```diff
@@ -6,7 +6,7 @@ AnimatorController:
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
-  m_Name: Item Pickup e
```

Comment:
@pjumruspun please recheck this I'm not sure what it is.

Reply:
I'll delete it, this was one of my attempts to fix the animator bug where I solved with creating an empty state and delete it instead